### DISCUSSION TITLE
Try change layout on recording use getstream.io

### DISCUSSION THREAD

**Discussion by teomikitenko** [supabase/supabase#26401] [Questions] [answered]
I using react sdk and create videochat but have one problem. I recording call and take this calls from storage - and this video recording dont have any styles(i need grid layout and change background). So i start searching solution in documentation on getstream.io and find it but only for server side. Solution is add some style in layout option :
`client.video.updateCallType(callTypeName, {
  settings: {
    recording: {
      mode: VideoRecordSettingsRequestModeEnum.AVAILABLE,
      audio_only: false,
      quality: VideoRecordSettingsRequestQualityEnum._1080P,
      layout: {
        name: VideoLayoutSettingsNameEnum.SPOTLIGHT,
        options: layoutOptions,
      },
    },
  },
});`
but react sdk client dont have option layout - client from react sdk just have mode,audio and quality.I create client like this:
`const client = new StreamVideoClient({ apiKey, user, token });`
Im using next js and in server side i just create token, hovewer in client side i create client and work with react sdk - so my answer is how change layout in recording call using react sdk

**Comment by silentworks** [comment, marked as answer]
This is better asked on a getstream.io forum as its not a Supabase issue. Please try their forum or Stack Overflow and you will get a better answer there.
